examples/ai_society/role_playing.py

```diff
@@ -69,4 +72,4 @@ def main(model_type=None) -> None:
 
 
 if __name__ == "__main__":
-    main()
+    main(model_type=ModelType.GPT_3_5_TURBO_16K)
```

Comment:
Why not going with None so that the default is used for all agents?

Reply:
I changed that to have a quick test on examples 😂 Just revert the change